.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,22 +1,37 @@
+-->
 
-## Explanation
-<!--
-  - Explain what your PR does. If this PR fixes an existing bug, please include
-  - "Fixes #bugnum:" in the explanation so that GitHub can auto-close the issue
-  - when this PR is merged.
-  -->
+1. This PR fixes or fixes part of #[fill_in_number_here].
+2. This PR does the following: [Explain here what your PR does and why]
 
-## Checklist
+## Essential Checklist
 <!-- Please tick the relevant boxes by putting an "x" in them. -->
-- [ ] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
-- [ ] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
+- [ ] The PR title and explanation starts with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
+- [ ] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have rationale included in the PR description.
 - [ ] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
-- [ ] The PR does not contain any unnecessary auto-generated code from Android Studio.
-- [ ] The PR is made from a branch that's **not** called "develop".
-- [ ] The PR is made from a branch that is up-to-date with "develop".
-- [ ] The PR's branch is based on "develop" and not on any other branch.
-- [ ] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.
+- [ ] The PR does not contain any unnecessary code changes from Android Studio.
+- [ ] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
+- [ ] The PR is **assigned** to appropriate reviewers. [Reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)
+
+## Proof that changes are correct
+
+<!--
+Add videos/screenshots of the user-facing interface in various display sizes (mainly phone and tablet) in both LTR and RTL mode to demonstrate that the changes made in this PR work correctly.
+[Oppia Android RTL Guide](https://github.com/oppia/oppia-android/wiki/RTL-Guidelines).
+-->
+
+## For UI-specific PRs only
+<!-- Delete this section if this PR does not include UI-related changes. -->
+
+### Espresso Test
+<!-- Add screenshots showing that all relevant Espresso tests are passing locally. -->
+
+### Accessibility Checks
+<!-- Explain how this PR has been tested for accessibility, or why doing so is not applicable. For more info, check out the [Oppia Android Accessibility Guide](https://github.com/oppia/oppia-android/wiki/Accessibility-(A11y)-Guide). -->
+
+### How to test this PR?
+<!-- Explain what exact steps reviewer should perform to test this PR. -->
```

Comment:
Please provide a list of the exact steps that the reviewer should perform in order to test this PR.

Reply:
Done.